bc_obps/registration/fixtures/dummy/document.json

```diff
@@ -1,24 +1,10 @@
 [
```

Comment:
can we please use something other than `dummy`? `mock` or some other suggestions here: https://www.khronos.org/about/inclusive-language#dummy

Reply:
Yikes. I'm very sorry, I didn't realize that "dummy" is an exclusionary term. Thanks for letting me know!
I've replaced it with `mock`